src/mjolnir/way_edges_processor.cc

```diff
@@ -0,0 +1,71 @@
+#include "mjolnir/way_edges_processor.h"
+
+#include <fstream>
+
+#include "baldr/edgeinfo.h"
+
+namespace valhalla {
+namespace mjolnir {
+
+namespace {
```

Comment:
One final nitpick: can you isolate the anonymous namespace, i.e. not have it nested inside valhalla/mjolnir?

Reply:
You're right they're no deal breakers, I'll approve so we leave it to you @Amr-Shams to decide if you want to do it :smile: